src/components/settings/modelProvider.tsx

```
@@ -1017,11 +1018,69 @@ const ModelProvider = () => {
+            </div>
+            <div className="my-8">
+              <input
+                type="number"
+                min="1"
+                max="100"
+                className="px-16 py-8 w-64 bg-surface1 hover:bg-surface1-hover rounded-8"
+                value={maxPastMessages}
+                onChange={(e) => {
+                  const value = parseInt(e.target.value)
+                  if (!isNaN(value) && value >= 1 && value <= 100) {
+                    settingsStore.setState({ maxPastMessages: value })
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)